config/config.go

```diff
@@ -166,6 +167,12 @@ type Kafka struct {
 	Topic   string
 }
 
+// PubSub configures access to Google Cloud Pub/Sub for task event reading/writing.
+type PubSub struct {
+	Topic   string
+	Project string
```

Comment:
This should also have `CredentialsFile string` to be consistent with the other google components.

Reply:
Fixed in https://github.com/ohsu-comp-bio/funnel/pull/465/commits/8a7743b41a4f0e9bffb88fcb6be190bbb651efc5

Reply:
It doesn't look like your `NewPubSubWriter` and `ReadPubSub` methods make use of the new field.